File: 1421-FindNumbersWithEvenNumberOfDigits/1421-FindNumbersWithEvenNumberOfDigits.js
// Last updated: 4/29/2026, 10:24:29 AM
/**
 * @param {number[]} nums
 * @return {number}
 */
// 1:15s
// O(n) time | O(1) space
var findNumbers = function(nums) {
    let count = 0;
    for(const num of nums) {
        const str = num.toString();
        if(str.length % 2 === 0) {
            count++;
        }
    }
    return count;
};

File: 1421-FindNumbersWithEvenNumberOfDigits/1421-FindNumbersWithEvenNumberOfDigits.py
# Last updated: 4/29/2026, 10:24:25 AM
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        count = 0
        for num in nums:
            num_len = len(str(num))
            if num_len % 2 == 0:
                count += 1
        
        return count